lib/confluence/metric_computer_runner.es6.js

```diff
@@ -86,10 +86,14 @@ foam.CLASS({
                 this.Release.BROWSER_NAME,
                 this.ArraySink.create()))
             .then((groups) => {
-          return groups.groupKeys.map((bName) => {
-            return groups.groups[bName].array[0];
-          });
-        });
+              return groups.groupKeys.map((bName) => {
```

Comment:
FWIW this had me wondering just a little bit if I might be inside some kind of sorting method, and if there's an `aName`. But I think this is the browser name. Expanding it wouldn't hurt.